src/components/TradingviewChart/Tooltip.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Skeleton from 'react-loading-skeleton';
+import { Box } from 'rebass';
+import styled from 'styled-components';
+
+import { useDarkModeManager } from '../../contexts/LocalStorage';
+import { formattedNum } from '../../utils';
+
+const Value = styled.div`
+  font-size: 22px;
+  color: ${({ theme }) => theme.text1};
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+`;
+
+const Title = styled.div`
+  font-size: 16px;
+  color: ${({ theme }) => theme.text1};
+  margin: 4px 0px;
+`;
+
+const Container = styled.div`
+  font-weight: 500;
+  left: -4px;
+  top: -50px;
+  background-color: transparent;
+`;
+
+export default function Tooltip({ type, base, title, useWeekly, baseChange, dateStr, price }) {
```

Comment:
Since it's a `.tsx` file, shoule we add the types for the properties? Or we will refactor all the components once we merge the PR for the typescript migration?

Reply:
Yes will add now only. Good point